gui/wxpython/datacatalog/tree.py

```diff
@@ -1498,6 +1497,7 @@ def SwitchMapset(self, grassdb, location, mapset, show_confirmation=False):
         """
         Switch to location and mapset interactively.
         """
+
```

Comment:
Not many programmers know this, but Python does not require an empty line between every two statements.